src/alexandria3k/data_sources/issn_subject_codes.py

```diff
@@ -131,19 +132,24 @@ def fetch_subject_codes(self, writer, issns):
             except (KeyError, ValueError):
                 warn(f"Error processing ISSN {issn}")
 
-    def download(self, database, data_location, sql_query=None):
+    def download(self, data_location, database=None, sql_query=None):
         """
-        Create a CSV file with ISSNs and their corresponding ASJC subject codes from API calls.
+        Create a CSV file with ISSNs and their corresponding ASJC subject codes from API calls
+        using the Scopt API.
+
+        :param data_location: The path specifying the CSV file where
+        the downloaded data will be stored.
+        :type data_location: str
 
         :param database: The path specifying the SQLite database to use in order to fetch the ISSNs.
         :type database: str
 
-        :param data_location: The path specifying the CSV file to use for the population.
-        :type data_location: str
-
-        :param sql_query: The SQL query to use in order to fetch the ISSNs, defaults to `None`.
-            The default query fetches all unique ISSNs from the database. The query should
-            return a single column with the name `issn`.
+        :param sql_query: The SQL query that will supply the ISSNs to fetch,
+        from the database specified, where the ISSNs are represented by `issn_print`
+        and `issn_electronic`, defaults to `None`.
```

Comment:
Please remove "where the ISSNs are represented by `issn_print` and `issn_electronic`" as this is not a required part of the specified interface. Please also clarify that the ISSN shall be formatted as 8 alphanumeric characters (.e.g `2249782X`) without any spaces or dashes.